src/KillbillClient.php

```diff
@@ -62,7 +62,7 @@ public function __construct(
         $password = 'password'
     ) {
         $this->logger = $logger ?: new NullLogger();
-        $this->configuration = Configuration::getDefaultConfiguration()
```

Comment:
Should we remove it entirely?

I see it still being used in places (e.g. `Configuration::getDefaultConfiguration()->getTempFolderPath()` in `ObjectSerializer`) -- this could get confusing.

Reply:
ObjectSerializer is part of generated code (everything is generated under `lib/`).
We could remove `Configuration::getDefaultConfiguration` from there by adding this file to swagger-ignore, but we'll have to maintain 300 lines of generated code.
Since only getTempFolderPath is involved, maybe it's not that critical?

Reply:
Ah, indeed, good point!